tasks/main.yml

```diff
@@ -16,6 +16,8 @@
   ansible.builtin.include_tasks: ./websrv_install.yml
 
 - name: Install certificates
+  #tags:
```

Comment:
Commented out code should be removed or used if still needed.

Reply:
Done